crates/cli/src/opts/rpc.rs

```diff
@@ -114,6 +114,16 @@ pub struct EtherscanOpts {
     #[serde(rename = "etherscan_api_key", skip_serializing_if = "Option::is_none")]
     pub key: Option<String>,
 
+    /// The Etherscan API version.
+    #[arg(
+        short,
+        long = "etherscan-api-version",
+        alias = "api-version",
+        env = "ETHERSCAN_API_VERSION"
+    )]
+    #[serde(rename = "etherscan_api_version", skip_serializing_if = "Option::is_none")]
+    pub api_version: Option<String>,
```

Comment:
nit: since the package is included here you could just use the struct and it should serialize/deseralize correctly.

Reply:
good point, yeah, could also get rid of the newly introduced InvalidApiVersion, requires https://github.com/foundry-rs/block-explorers/pull/85

Reply:
Simplified inhttps://github.com/foundry-rs/foundry/pull/10440/commits/24782b1c90150d360d10258381df60cf695d7027